reference/conanfile/tools/gnu/pkgconfigdeps.rst

```diff
@@ -181,6 +181,29 @@ Their ``pkg_config_name`` properties are used as the final PC file names:
     Requires: zlib
 
 
+A special mention when a component shares the same final PC name as the root package one:
```

Comment:
Is PC a frequent way to refer to pkg-config ? I think it's better to directly say `pkg-config`

Reply:
I'm referring to any `****.pc` file

Reply:
Call it "``.pc`` file name" for clarity maybe